python/pylibvw.cc

```diff
@@ -546,31 +546,101 @@ std::string varray_char_to_string(v_array<char>& a)
 
 std::string my_get_tag(example_ptr ec) { return varray_char_to_string(ec->tag); }
 
-uint32_t ex_num_namespaces(example_ptr ec) { return (uint32_t)ec->indices.size(); }
+uint32_t ex_num_namespaces(example_ptr ec) { return ec->feature_space.end() - ec->feature_space.begin(); }
```

Comment:
This does not need to change.
```suggestion
uint32_t ex_num_namespaces(example_ptr ec) { return (uint32_t)ec->indices.size(); }
```